plugins/transport-nio/build.gradle

```diff
@@ -19,12 +19,15 @@ import org.elasticsearch.gradle.info.BuildParams
  * under the License.
  */
 apply plugin: "elasticsearch.publish"
+apply plugin: 'elasticsearch.internal-cluster-test'
 
 esplugin {
   description 'The nio transport.'
   classname 'org.elasticsearch.transport.nio.NioTransportPlugin'
 }
 
+// integTest have been converted to internalClusterTest 
+integTest.enabled = false
```

Comment:
I thought we ditched the need for this?

Reply:
This one is special such neither javaRestTest or yamlRestTest is applied here (only internalClusterTest). The auto-disable only works from the java/yamlRestTest application since there are some other projects with existing `internalClusterTest` and don't want to auto disable integTest for those. These should be pretty rare, and can be deleted once integTest is gone.